canvas_sdk/v1/data/note.py

```diff
@@ -0,0 +1,152 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from canvas_sdk.v1.data.patient import Patient
+
+# from canvas_sdk.v1.data.staff import Staff
+from canvas_sdk.v1.data.user import CanvasUser
+
+
+class NoteType(models.Model):
+    """NoteType."""
+
+    class NoteTypeCategories(models.TextChoices):
```

Comment:
I've been putting these outside of the class, at the module level. It's a bit less verbose that way, and aside from organization (which you get from the module anyway), I'm not sure there's any benefit to having them inside the class.

Reply:
Done.